src/api2/managers/wallets-manager.js

```diff
@@ -173,8 +173,17 @@ export class WalletsManager {
       ]
     })
 
+    const walletWithSession = new Wallet(
+      mainWallet.email,
+      mainWallet.keypair,
+      mainWallet.accountId,
+      mainWallet.id,
+      response.data.session.id,
+      response.data.session.encryptionKey
```

Comment:
use lodash's `get` here pls. session fetched by include, theoretically, the user may omit including of it